Frameworks/UIKit/UIImage.mm

```diff
@@ -893,10 +720,11 @@ - (void)setOrientation:(UIImageOrientation)orientation {
 */
 - (CGSize)size {
     CGSize size;
+    CGImageRef image = getImage(self);
 
-    if (getImage(self)) {
-        size.width = float(getImage(self)->Backing()->Width());
-        size.height = float(getImage(self)->Backing()->Height());
+    if (image) {
+        size.width = float(CGImageGetWidth(image));
```

Comment:
out of scope for this CR.
There should be a UIImage cleanup at somepoint by the UI Team.

---
In reply to: [84550720](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84550720) [](ancestors = 84550720)